src/test/java/org/broadinstitute/hellbender/utils/IndexUtilsUnitTest.java

```diff
@@ -89,43 +95,94 @@ public void testLoadIndexAcceptOldIndex() throws Exception {
         final File tmpVcf= tmpFeatureFilePath.toFile();
         Thread.sleep(1000L); //wait a second
         tmpFeatureFilePath.toFile().setLastModified(System.currentTimeMillis()); //touch the file but not the index
```

Comment:
You added a helper method to do this, you might as well use it here for this old code as well